workflow/hpxml_inputs.json

```diff
@@ -1107,6 +1113,9 @@
     "hot_water_distribution_recirc_control_type": "presence sensor demand control",
     "hot_water_distribution_pipe_r": 3
   },
+  "sample_files/base-dhw-recirc-demand-scheduled.xml": {
+    "parent_hpxml": "sample_files/base-dhw-recirc-demand.xml"
+  },
```

Comment:
Maybe we just make a single `base-schedules-simple2.xml` that has temperature dependent fridge schedules and a scheduled recirculation pump?